packages/ppx_string_interpolation/ppx_string_interpolation.1.0.0/opam

```diff
@@ -0,0 +1,30 @@
+opam-version: "2.0"
+name: "ppx_string_interpolation"
+synopsis: "String interpolation PPX preprocessor"
+description: "
+String interpolation PPX preprocessor.
+"
+maintainer: "Konstantin Romanov <konstantinsromanov@gmail.com>"
+authors: ["Konstantin Romanov <konstantinsromanov@gmail.com>"]
+homepage: "https://https://github.com/bloomberg/ppx_string_interpolation"
```

Comment:
I'm not reviewer and not maintainer, and I'm not going to disturb the review process,
but just for your information.
Is this typo?
`https://https://github.com/...`  (https:// twice)

Reply:
Thanks a lot!